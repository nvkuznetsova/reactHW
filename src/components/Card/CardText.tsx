import React from 'react';

export type CardTextProps = {
  cardText: string,
};

export const CardText: React.FC<CardTextProps> = ({ cardText }) => (
  <p className="card-text">{cardText}</p>
);
