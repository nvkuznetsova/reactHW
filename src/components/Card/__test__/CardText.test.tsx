import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';

import { CardText, CardTextProps } from '../CardText';

describe('Card', () => {
  const getProps = (): CardTextProps => ({
    cardText: 'test',
  });
  const setup = (): ShallowWrapper => shallow(<CardText {...getProps()} />);

  it('should match snapshot', () => {
    const component = setup();
    expect(component).toMatchSnapshot()
  });
});