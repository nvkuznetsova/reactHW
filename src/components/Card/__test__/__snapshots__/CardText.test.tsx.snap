// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot 1`] = `
<p
  className="card-text"
>
  test
</p>
`;
